code/datums/components/shuttle_cling.dm

```diff
@@ -47,15 +47,17 @@
 /datum/component/shuttle_cling/proc/update_state()
 	SIGNAL_HANDLER
 
-	if(!is_on_hyperspace(parent))
+	var/atom/movable/movable = parent
+
+	if(!is_on_hyperspace(movable) || !isturf(movable))
```

Comment:
Is this check correct? When is it possible for a movable atom to be a turf- shouldn't this always cause a qdel+return?

Reply:
That should be movable.loc . I don’t understand this fix in the slightest but it works?? I’ll see if it works after I fix it as well